sdk/src/syscall/fees.rs

```diff
@@ -0,0 +1,71 @@
+//! This account contains the current cluster fees
+//!
+use crate::account::Account;
+use crate::account_utils::State;
+use crate::fee_calculator::FeeCalculator;
+use crate::pubkey::Pubkey;
+use crate::syscall;
+use bincode::serialized_size;
+
+/// "Sysca11Fees11111111111111111111111111"
+///  fees account pubkey
+const ID: [u8; 32] = [
+    6, 167, 211, 138, 69, 218, 104, 33, 3, 92, 89, 173, 16, 89, 109, 253, 49, 97, 98, 165, 87, 222,
+    119, 112, 253, 90, 76, 184, 0, 0, 0, 0,
+];
+
+pub fn id() -> Pubkey {
+    Pubkey::new(&ID)
+}
+
+pub fn check_id(pubkey: &Pubkey) -> bool {
+    pubkey.as_ref() == ID
+}
+
+#[repr(C)]
+#[derive(Serialize, Deserialize, Debug, Default)]
+pub struct Fees {
+    pub fee_calculator: FeeCalculator,
+}
+
+impl Fees {
+    pub fn from(account: &Account) -> Option<Self> {
+        account.state().ok()
+    }
+    pub fn to(&self, account: &mut Account) -> Option<()> {
```

Comment:
I don't really like the names I made up here...  I was thinking about a `with(FnMut)` thing at some point to cover these really short serialize/deserialize lifetimes